### PR TITLE
HPCC-17445 Document COMPRESSED for CSV

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>

@RichardTaylorHPCC Please review

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by RichardTaylorHPCC** [comment]
DATASET changes need more clarification

**Comment by JamesDeFabia** [comment]
@RichardTaylorHPCC Changes made, please look again

**Comment by RichardTaylorHPCC** [comment]
Syntax diagram and discussion table don't match. COMPRESSED discussion in the table follows ENCRYPT, so COMPRESSED should also follow ENCRYPT in the syntax diagram

**Comment by JamesDeFabia** [comment]
@RichardTaylorHPCC Edits made as requested, please look once again

**Comment by JamesDeFabia** [comment]
@RichardTaylorHPCC Made all the edits you requested, please have another look

File: docs/ECLLanguageReference/ECLR_mods/BltInFunc-OUTPUT.xml [hunk 1]
```diff
@@ -589,15 +589,15 @@ OUTPUT(People(Attr1=FALSE));
     role="bold">]</emphasis><emphasis> ,file
     </emphasis><emphasis>,</emphasis><emphasis role="bold"> CSV<indexterm>
         <primary>CSV</primary>
-      </indexterm> [ (</emphasis><emphasis>csvoptions</emphasis><emphasis
+      </indexterm>[ ,(</emphasis><emphasis>csvoptions</emphasis><emphasis
```
Comment (JamesDeFabia) [inline docs/ECLLanguageReference/ECLR_mods/BltInFunc-OUTPUT.xml]: Superfluous comma

**Comment by RichardTaylorHPCC** [comment]
OK

**Comment by RichardTaylorHPCC** [comment]
Looks OK

**Pull request merged**